src/support/action/debugGiven.js

```diff
@@ -0,0 +1,3 @@
+module.exports = () => {
+	browser.debug();
```

Comment:
We don't need to have 3 versions of this file, we should be able to have just 1 that we re-use - called something more generic like debug.js